### PR TITLE
feat(docs for testing constructs): Adding Testing Constructs from samples repo

### PR DESCRIPTION
**Pull request by MichaelWalker-git**
### Description
- add to the section regarding our samples repo and leveraging for the development process

Fixes #

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the project license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,57 @@ The following constructs are available in the library:
...
+
+
+### Step 1: Building the Generative AI CDK Construct
+
+Navigate to the [Generative AI CDK Construct Repository] (https://github.com/aws-samples/generative-ai-cdk-constructs-samples):
+    Open your command line interface and change directory to the generative AI CDK construct repository.
+
+- Execute the command ```yarn build```. 
+  - This command runs npx projen build and generates a .jsii file in your repository.
+![Npm_Build.png](docs%2FNpm_Build.png)
```
Comment (scottschreckengaust) [inline README.md:95]:
```suggestion
```

I would remove the images

File: README.md [hunk 2]
```diff
@@ -81,6 +81,57 @@ The following constructs are available in the library:
...
+
+### Step 2: Packaging the Constructs
+![JS_Package.png](docs%2FJS_Package.png)
```
Comment (scottschreckengaust) [inline README.md:98]:
```suggestion
```

this too

File: README.md [hunk 3]
```diff
@@ -81,6 +81,57 @@ The following constructs are available in the library:
...
+
+### Step 3: Integrating with Sample Application
+
+1. Drag and Drop the TGZ File:
+    - Drag and drop the .tgz file into your samples repository, ideally at the root like "samples/document_explorer".
+
+2. Modify package.json in Sample Repo:
+    - Open package.json in your samples repository.
+    - Under dependencies, locate the entry for "@cdklabs/generative-ai-cdk-constructs".
+    - Replace the existing entry with file:<path-to-tgz-file>. For example: file:generative-ai-cdk-constructs-0.0.0.jsii.tgz.
```
Comment (scottschreckengaust) [inline README.md:113]:
```suggestion
    - Replace the existing entry with file:<path-to-tgz-file>. For example: 
    ```
...
  "dependencies": {
    "@cdklabs/generative-ai-cdk-constructs": "file:generative-ai-cdk-constructs@0.0.0.jsii.tgz",
...
    ```
```

File: docs/JS_Package.png
Comment (scottschreckengaust) [inline docs/JS_Package.png:1]: no need to have an image

File: docs/Npm_Build.png
Comment (scottschreckengaust) [inline docs/Npm_Build.png:1]: no images

File: docs/OutputJS.png
Comment (scottschreckengaust) [inline docs/OutputJS.png:1]: no images

File: README.md [hunk 4]
```diff
@@ -81,6 +81,54 @@ The following constructs are available in the library:
...
+
+- Using the samples repository to test your locally changed constructs. This is also useful for development.
+
+
+### Step 1: Building the Generative AI CDK Construct
+
+Navigate to the [Generative AI CDK Construct Repository] (https://github.com/aws-samples/generative-ai-cdk-constructs-samples):
+    Open your command line interface and change directory to the generative AI CDK construct repository.
+
+- Execute the command ```yarn build```.
```
Comment (krokoko) [inline README.md:93]: you can do it through projen as well: projen build

**Comment by krokoko** [comment]
LGTM ! Maybe we could move it to the developer guide md file ?

**Review by krokoko — DISMISSED** [review]

**Review by scottschreckengaust — APPROVED** [review]

**Pull request merged**